app/Listeners/PersistTweetId.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Listeners;
+
+use App\Notifications\PostArticleToTwitter;
+use Illuminate\Notifications\Events\NotificationSent;
+
+final class PersistTweetId
+{
+    public function handle(NotificationSent $event): void
+    {
+        if (! $event->notification instanceof PostArticleToTwitter) {
+            return;
+        }
+
+        $event->notification
+            ->getArticle()
+            ->update([
+                'tweet_id' => $event->response->id,
+            ]);
```

Comment:
Same remark here as before. Ditch the false check and move the code inside the if statement.

```suggestion
        if ($event->notification instanceof PostArticleToTwitter) {
            $event->notification->article()->update([
                'tweet_id' => $event->response->id,
            ]);
        }
```